kubeflow/common/iap-ingress/base/config-map.yaml

```diff
@@ -167,11 +167,12 @@ data:
       echo health check URI is ${HEALTH_CHECK_URI}
 
       # See: https://cloud.google.com/service-mesh/docs/iap-integration#deploying_the_load_balancer
-      HC_INGRESS_PORT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="status-port")].nodePort}')
```

Comment:
This is instructed in https://cloud.google.com/service-mesh/docs/unified-install/options/iap-integration#deploying_the_load_balancer. Do you think this documentation need to be updated? And why? Can you parse the service yaml for istio-ingressgateway here?

Reply:
Reverted